bin/generate_tests.py

```diff
@@ -481,7 +492,7 @@ def generate_exercise(
             logger.debug(f"{slug}: formatting tmp file {tmpfile}")
             format_file(tmpfile)
         except subprocess.CalledProcessError as e:
-            return False
+            pass
```

Comment:
Shouldn't we at least print an error message here?

Reply:
I'm not sure that that's necessary because we already have the output of roc format. For me it always crashes when I've written part of the template but haven't deleted the json yet, so I had been commenting out the format command. Are there other scenarios you can think of where an additional error message would be helpful?

Reply:
Oh I didn't think of that, yes we already have a very long error message, no need to add one more. 👍